qiskit/visualization/pulse/matplotlib.py

```diff
@@ -281,15 +286,35 @@ def draw(self, pulse, dt, interp_method, scale=1, scaling=None):
 class ScheduleDrawer:
     """A class to create figure for schedule and channel."""
 
-    def __init__(self, style):
+    def __init__(self, style: 'SchedStyle'):
         """Create new figure.
 
         Args:
-            style (SchedStyle): style sheet
+            style: Style sheet for pulse schedule visualization.
         """
         self.style = style or SchedStyle()
 
-    def _build_channels(self, schedule, channels, t0, tf, show_framechange_channels=True):
+    def _build_channels(self, schedule: 'ScheduleComponent',
+                        channels: List['Channel'],
+                        t0: int, tf: int,
+                        show_framechange_channels: bool = True)\
```

Comment:
I think I usually see this in this situation:
```
       arg: type
) -> Tuple[...
```
with a newline before the `)` and no indentation

Reply:
ok, I'll update them.